### DISCUSSION TITLE
[Feature] Whether multiple Tracing link information can be synthesized

### DISCUSSION THREAD

**Discussion by wl2027** [apache/skywalking#12179] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

The largest unit of OpenTracing is a single link, but operations such as some workflows or state machines are often multiple link combinations. If you want to combine and observe this flow state, in addition to marking the key of the service on the link, is there any other way to combine multiple tracing into a simple and intuitive data flow? Similar to the processing log in the workflow, it is only the log of the processing link

The idea is that plug-ins do not monitor activiti workflows, that link observations on business processes are not suitable for skywalking, or that this should be made into BPM alone and not added to APM

是否可以实现多个Tracing链路信息合成

OpenTracing中最大单位是单个链路,但是像一些工作流或状态机这类的操作往往都是多个链路组合,如果想组合起来观测这种流转状态,除了在链路上标记业务的key,还有别能实现将多个tracing组合成简单直观数据流的方式吗?类似工作流中的办理日志这类,不过是办理链路的日志这样

有这种想法是看到并没有去监控activiti工作流的插件,对业务流程上的链路观测不知道适不适用于用skywalking,还是说这种应该单独做成BPM而不是加在APM上

### Are you willing to submit a pull request to implement this on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
I don't know the background of this asking.
SkyWalking reference from span to span supports mutiple parents.
